runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/FakeWindmillServer.java

```diff
@@ -485,4 +437,55 @@ public boolean isReady() {
   public void setIsReady(boolean ready) {
     this.isReady = ready;
   }
+
+  static class ResponseQueue<T, U> {
```

Comment:
Is it preferred style to put inner classes later? Otherwise reduce churn by keeping this where it was.

Reply:
it is preferred style (at least in Google style guide), but this was fixed by the linter (spotless apply) since I modified the file.

My only changes were renaming/reimport GetWorkStream since it was moved to the WindmillStream.java.